src/test/java/com/hotels/beeju/extensions/HiveServer2JUnitExtensionTest.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (C) 2015-2019 Expedia, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hotels.beeju.extensions;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.apache.hadoop.hive.conf.HiveConf;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+public class HiveServer2JUnitExtensionTest {
+
+  private static final String DATABASE = "my_test_db";
+
+  @RegisterExtension
+  HiveServer2JUnitExtension server = new HiveServer2JUnitExtension(DATABASE);
+
+  @Test
+  public void defaultDatabaseName() {
+    HiveServer2JUnitExtension server = new HiveServer2JUnitExtension(DATABASE, null);
```

Comment:
Is this needed here? Because the test passes without it too.

Reply:
+1 it's unused